README.md

```diff
@@ -257,7 +226,7 @@ If you like this project...
 
 > Give credit to everyone who inspired your codebase.
 
-I would like to thank...
+I would like to thank...[Tasty API](https://rapidapi.com/apidojo/api/tasty/) for providing the data
```

Comment:
You have rightly acknowledged the source of the data. Kindly also give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/) and as stated in the [requirements of the project](https://github.com/microverseinc/curriculum-react-redux/blob/5801583dd2ad132a9326d4bd429291d954e4e824/capstone/react_capstone.md?plain=1#L63).